src/main/java/org/rust/lang/core/psi/RustTokenElementTypes.java

```diff
@@ -54,7 +54,7 @@
     RustTokenType SUPER = new RustKeywordTokenType("super");
     RustTokenType TRAIT = new RustKeywordTokenType("trait");
     RustTokenType TRUE = new RustKeywordTokenType("true");
-    RustTokenType TYPE = new RustKeywordTokenType("type");
+    RustTokenType TYPE_KW = new RustKeywordTokenType("type");
```

Comment:
>Though, unfortunately, this would be breaking change.

No, this would be just refactoring =) We have a solid test suite and a nice Kotlin type system, so even huge refactorings are not a problem. 

>What do you think about prefixing all keyword token types with KW_, e.g. KW_TYPE, KW_UNSIZED? 

I think that while `TYPE` is a single exceptional case it is fine to deal with it specially. I personally prefer `WHILE` to `KW_WHILE` in parser code, so while we don't have at least `n` collisions, I think we can live without `KW_`. 

>The same idea might be applied to operators - OP_xxx.

I've been thinking recently about changing `LPAREN` and friends to `'('` alternatives which are more readable.